lib/rules/declaration-property-unit-allowed-list/README.md

```diff
@@ -83,3 +83,34 @@ a { animation-duration: 5s; }
 ```css
 a { line-height: 1; }
 ```
+
+## Optional secondary options
+
+### `ignore: ["inside-function"]`
+
+Ignore units that are inside a function.
+
+For example, given:
+
+```
+{
+  '/^border/': ['px'],
+  '/^background/': ['%'],
+}
+```
+
+The following patterns are _not_ considered violations:
+
+<!-- prettier-ignore -->
+```css
+button {
+  button { border: 1px solid hsla(162deg, 51%, 35%, 0.8); }
+}
+```
+
+<!-- prettier-ignore -->
+```css
+button {
```

Comment:
[suggestion] How about using the `a` type selector for consistency?

````suggestion
a {
  border: 1px solid hsla(162deg, 51%, 35%, 0.8);
}
```

<!-- prettier-ignore -->
```css
a {
````

See also <https://github.com/stylelint/stylelint/blob/d4881680bbcca06d58aa6d4f48806e09a949c96e/docs/developer-guide/rules.md#write-the-readme>.

Reply:
I was using `button` since the properties seemed more applicable to that, but `a` is fine.